src/main/java/gregtech/core/hooks/SoundHooks.java

```diff
@@ -0,0 +1,41 @@
+package gregtech.core.hooks;
+
+import gregtech.api.items.metaitem.MetaItem;
+import gregtech.api.items.metaitem.MusicDiscStats;
+import gregtech.api.items.metaitem.stats.IItemBehaviour;
+import net.minecraft.client.multiplayer.WorldClient;
+import net.minecraft.client.renderer.RenderGlobal;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemRecord;
+import net.minecraft.util.math.BlockPos;
+import net.minecraftforge.fml.common.ObfuscationReflectionHelper;
+
+public class SoundHooks {
+
+    public static void playRecord(RenderGlobal renderGlobal, int type, BlockPos pos, int data) {
+        if (type == 1010) {
+            WorldClient world = ObfuscationReflectionHelper.getPrivateValue(RenderGlobal.class, renderGlobal, "world");
```

Comment:
(1) use the srg name.
(2) would better pass the world as a perameter from the RenderGlobalVisitor. (It's a little expensive to use reflection for repetitive things)

Reply:
Given that people won't be spamming jukeboxes with an unobtainable item that doesn't even appear in JEI, I don't think this would be a problem.

Reply:
Although I did fix the srg thing: it's field_72769_h.